cpp/tests/utilities/column_utilities.hpp

```diff
@@ -50,5 +52,39 @@ void expect_columns_equal(cudf::column_view lhs, cudf::column_view rhs);
 void expect_equal_buffers(void const* lhs, void const* rhs,
                           std::size_t size_bytes);
 
+/**
+ * @brief Copies the data and bitmask of a `column_view` to the host.
+ *
+ * @tparam T The data type of the elements of the `column_view`
+ * @param c the `column_view` to copy from
+ * @return std::pair<std::vector<T>, std::vector<bitmask_type>> first is the
+ *  `column_view`'s data, and second is the column's bitmask.
+ */
+template <typename T>
+std::pair<std::vector<T>, std::vector<bitmask_type>> to_host(column_view c) {
+  std::size_t const mask_bytes{bitmask_allocation_size_bytes(c.size())};
+  std::vector<T> host_data;
+  std::vector<bitmask_type> host_bitmask;
+
+  auto col = column(c);
+
+  if (col.size() > 0) {
+    host_data.resize(col.size());
+    CUDA_TRY(cudaMemcpy(host_data.data(), col.view().head<T>(),
+                        col.size() * sizeof(T),
+                        cudaMemcpyDeviceToHost));
+  }
+
+  if (col.nullable()) {
+    size_t num_mask_elements = 
```

Comment:
I believe @jrhemstad is working on a utility function for this.

Reply:
Should I wait on that to merge this?

Reply:
`num_bitmask_words` was already merged. You can update this PR to use it.